go.mod

```diff
@@ -49,48 +49,48 @@ require (
 	github.com/hashicorp/raft v1.3.2-0.20210825230038-1a621031eb2b
 	github.com/hashicorp/raft-boltdb v0.0.0-20171010151810-6e5ba93211ea
 	github.com/im7mortal/kmutex v1.0.1
-	github.com/juju/ansiterm v0.0.0-20210929141451-8b71cc96ebdc
+	github.com/juju/ansiterm v1.0.0
 	github.com/juju/blobstore/v2 v2.0.0-20220204135513-4876189e56a4
 	github.com/juju/charm/v8 v8.0.0-20220509231111-ed6d505a46f4
```

Comment:
charm/v8 has a v8.0.0 tag which points to an older commit
could we add a new v8.0.1 tag and use that here